tasks/main.yml

```diff
@@ -14,3 +19,18 @@
   tags:
     - consul_keystore_configure
   when: consul_keystore_name is defined
+
+- name: Consul | Delete diff keys
```

Comment:
wouldn't [manage_keys_yml_file.yml](https://github.com/idealista/consul_keystore_role/pull/16/files#diff-e56df6b67f8e9511b9354f9e6de4acf7336fbc5c9c215f9dbf570a7ce05a61e9) be a more suitable place to do this logic?

Reply:
Yes, totally true. The issue there is this file is looped in main and this task must be executed after all looped executions. If I put this task there, it will be executed on every loop with weird results.

Reply:
mmm that's true, did not see the loop. Alright then :smiley: